osu.Game/Screens/Play/Player.cs

```diff
@@ -435,9 +435,14 @@ private Drawable createUnderlayComponents(WorkingBeatmap working)
                 RelativeSizeAxes = Axes.Both,
                 Children = new Drawable[]
                 {
+                    new LegacyComboFire(ScoreProcessor)
```

Comment:
legacy elements cannot be in the player class. this should be skihnnable and the default should be "none" for non-legacy skins until a better visualisation is envisaged.